content/en/profiler/enabling/dotnet.md

```diff
@@ -53,18 +53,17 @@ Supported languages
 
-|      Feature         | Required `dd-trace-dotnet` version      | Required .NET Runtime versions        |
-|----------------------|-----------------------------------------|---------------------------------------|
-| Wall time profiling        | 2.7.0+                     |All supported runtime versions.      |
-| CPU profiling        | 2.15.0+                       | All supported runtime versions.      |
-| Exceptions profiling        | 2.31.0+                       | All supported runtime versions.      |
-| Allocations profiling        | beta, 2.18.0+                       | .NET 6+      |
-| Lock Contention profiling        | 2.31.0+                       | .NET 5+      |
-| Live heap profiling        | beta, 2.22.0+                       | .NET 7+      |
-| [Code Hotspots][12]        | 2.7.0+                       | All supported runtime versions.      |
-| [Endpoint Profiling][13]            | 2.15.0+                       | All supported runtime versions.      |
-| Timeline            | 2.30.0+                       | All supported runtime versions (except .NET 5+ required for garbage collection details).     |
-| Azure App Services (public beta) | 2.39.0+                       | All supported runtime versions.   |
+| Feature                   | Required `dd-trace-dotnet` version | Required .NET Runtime versions                                                           |
+|---------------------------|------------------------------------|------------------------------------------------------------------------------------------|
+| Wall time profiling       | 2.7.0+                             | All supported runtime versions.                                                          |
+| CPU profiling             | 2.15.0+                            | All supported runtime versions.                                                          |
+| Exceptions profiling      | 2.31.0+                            | All supported runtime versions.                                                          |
+| Allocations profiling     | beta, 2.18.0+                      | .NET 6+                                                                                  |
+| Lock Contention profiling | 2.31.0+                            | .NET 5+                                                                                  |
+| Live heap profiling       | beta, 2.22.0+                      | .NET 7+                                                                                  |
+| [Code Hotspots][12]       | 2.7.0+                             | All supported runtime versions.                                                          |
+| [Endpoint Profiling][13]  | 2.15.0+                            | All supported runtime versions.                                                          |
+| Timeline                  | 2.30.0+                            | All supported runtime versions (except .NET 5+ required for garbage collection details). |
```

Comment:
I just want to confirm that you intended to remove the Azure App Services row entirely, versus just removing the beta verbiage.

Reply:
I confirm. This is not a feature, having it in this list only made sense when the support was in beta.